src/main/java/net/sf/jabref/model/entry/BibtexEntry.java

```diff
@@ -484,4 +485,64 @@ public String getPublicationDate() {
         }
         return year;
     }
+
+    public void putKeywords(ArrayList<String> keywords) {
+        // Set Keyword Field
```

Comment:
use Objects.requireNonNull(keywords) at the beginning to ensure that a null value is detected early on.